packages/block-editor/package.json

```diff
@@ -54,10 +55,11 @@
 		"@wordpress/wordcount": "file:../wordcount",
 		"classnames": "^2.2.5",
 		"css-mediaquery": "^0.1.2",
+		"css-tree": "^1.0.0-alpha.39",
 		"diff": "^4.0.2",
 		"dom-scroll-into-view": "^1.2.1",
-		"inherits": "^2.0.3",
 		"lodash": "^4.17.19",
+		"ltrim": "^1.0.0",
```

Comment:
Adding a package for such a small thing is not a great idea IMO. Use regular string functions if possible.

Reply:
Apparently the package wasn't even in use, so it has been removed now.